frontend/src/components/Comment/CommentList.js

```diff
@@ -1,43 +1,36 @@
 import React from 'react'
-import Comment from './Comment'
-import { getCommentByPicture } from './CommentAPI';
+import { useState, useEffect } from 'react'
+import { getCommentByPicture } from './CommentAPI'
+import {Comment} from './Comment'
+import './SCSS/CommentList.scss'
 
-class CommentList extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            commentList: []
-        }
-    }
-    componentDidMount() {
-        this.loadComments();
-    }
+const CommentList = (props) => {
 
-    componentDidUpdate(prevProps, prevState, snapshot) {
-        if(prevProps.postId !== this.props.postId) {
-            this.loadComments();
-        }
-    }
+    const [commentList, setcommentList] = useState([])
```

Comment:
Just a nitpicky comment: `comment` needs a capital C in your setter >> `setCommentList`